.github/workflows/ci.yml

```diff
@@ -0,0 +1,49 @@
+name: CI
+
+on:
+  push:
+
+jobs:
+  arduino:
+    name: Arduino Compile
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        board:
+          - adafruit:samd:adafruit_itsybitsy_m4
+          - teensy:avr:teensy36
```

Comment:
ci workflow will attempt to run a build on this list of boards using the `arduino-cli` as a basic verification that the code is not broken.